### PR TITLE
feat: variant with type number in the payload

### PR DESCRIPTION
**Pull request by andreas-unleash**
Adds a variant test with number as payload type

### CODE REVIEW

**Review by kwasniew — APPROVED** [review]
I'd also add strategy variant example with number since it's our future direction

**Review by sighphyre — COMMENTED** [review]
Be cool if we could also include a "value" : "1" or something equivalently integer-y. I'd like to be able to signal to consumers that the decimal isn't required

File: specifications/08-variants.json [hunk 1]
```diff
@@ -205,6 +205,27 @@
                         }
                     }
                 ]
+            },
+            {
+                "name": "Feature.Variants.H",
+                "description": "variant with payload number",
```
Comment (sighphyre) [inline specifications/08-variants.json:211]: I think this signals the intent better

```suggestion
                "description": "variant with numeric payload",
```

File: specifications/08-variants.json [hunk 2]
```diff
@@ -430,6 +451,21 @@
                 "name": "disabled",
                 "enabled": false
             }
+        },
+        {
+            "description": "Feature.Variants.H should return variant1 with a number in the payload",
```
Comment (sighphyre) [inline specifications/08-variants.json:456]: Same same

```suggestion
            "description": "Feature.Variants.H should return variant1 with numeric type payload",
```

**Comment by andreas-unleash** [comment]
Cool I will add that to the strategy variants tests

**Pull request merged**